packages/shared/src/hooks/useExtensionPermission.ts

```diff
@@ -18,7 +18,7 @@ export type ContentScriptStatus = () => {
 
 interface ExtensionPermission {
   useContentScriptStatus: ContentScriptStatus;
-  requestContentScripts: () => Promise<boolean>;
+  requestContentScripts: (skipRedirect?: boolean) => Promise<boolean>;
```

Comment:
Better maybe to add it as object param

```ts
interface ExtensionPermission {
  useContentScriptStatus: ContentScriptStatus;
  requestContentScripts: ({ skipRedirect }?: { skipRedirect: boolean }) => Promise<boolean>;
```

this will allow us more customisability in the future when adding more options.